### ISSUE TITLE
Home -> SiteMapFooter Inaccurate Link Text 

### ISSUE THREAD

**Issue by johnhbenetech** [benetech/MathShare#774]
In the list of links with the title “Info”, three links all have the same URL: “Privacy Policy”, “FERPA/COPPA” and “Terms of Service”.  They all link to the /privacy route, but when each link is clicked, the exact same information (a short privacy policy and link to the Benetech website) is shown. 
 
Suggested Solution 
 
Ensure that link text accurately describes the purpose and destination of the control.  Here, it may be the case that the content of the destination page is not yet complete.  If so, it should be noted that when the information is in place, each link should navigate to a specific page section rather than the page as a whole.  Otherwise, these links should be removed, or their destinations and/or text should be updated.

**Comment by rupeshparab** [comment]
@johnhbenetech 
Please let me know if there were be separate pages for these links, or if some of these links need to be removed

**Comment by johnhbenetech** [comment]
@benetechMatt please advise

**Comment by alexrcabral** [comment]
Matt this is up to you. I'm of the opinion that they should continue linking to the same page, but to a different header within the same page. 

There's not enough content / difference in topic in each to require entirely different pages, but they are things that people will want to be linked directly to.

**Comment by johnhbenetech** [comment]
@benetechMatt Thoughts on this?

**Comment by benetechMatt** [comment]
remove “FERPA/COPPA” and “Terms of Service” links in the footer at this time.  When we have content for these pages we will add them again

**Comment by benetechMatt** [comment]
completed

**Comment by jscholes** [comment]
Fixed in https://github.com/benetech/MathShare/commit/d407c7decdb7a16cca05d9f050be34edb244e345.
